rotkehlchen/tests/api/test_accounting_rules.py

```diff
@@ -530,25 +532,47 @@ def test_cache_invalidation(rotkehlchen_api_server: APIServer):
 @pytest.mark.parametrize('initialize_accounting_rules', [True])
 def test_import_export_accounting_rules(rotkehlchen_api_server: 'APIServer'):
     """Test that exporting and importing accounting rules works fine."""
+    async_query = random.choice([True, False])
     with rotkehlchen_api_server.rest_api.rotkehlchen.data.db.conn.read_ctx() as cursor:
         initial_rules = cursor.execute(
             'SELECT * FROM accounting_rules WHERE identifier IN (1, 82);',
         ).fetchall()
         initial_properties = cursor.execute('SELECT * FROM linked_rules_properties').fetchall()
 
     with TemporaryDirectory() as temp_directory:
+        response = requests.post(
+            api_url_for(  # export the accounting rules as a response json
+                rotkehlchen_api_server,
+                'accountingrulesexportresource',
+            ), json={'async_query': async_query},
+        )
+        if async_query:
+            response_result = wait_for_async_task_with_result(
+                server=rotkehlchen_api_server,
+                task_id=response.json()['result']['task_id'],
+            )
+        else:
+            response_result = assert_proper_response_with_result(response)
```

Comment:
have you checked if this structure is something common around the code? Maybe we could have a `check_async_request` that does this. YOu are repeting code